sp-files/.sphinx/.markdownlint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "default": false,
+    "MD003": { "style": "atx" },
+    "MD013": { "code_blocks": false, "tables": false, "stern": true, "line_length": 80},
```

Comment:
The only reason I see for lines to be limited to 80 characters wide would be for print format (A4 supports 80 monospaced characters across at default font size). If we're not doing it for code blocks, I don't particularly see the point of the arbitrary limit for lines in general.

Best readability is something like 66 characters across. 80 is a holdover from the above - and the fact that original Unix terminals were 80x25 characters. There are *very* few applications that still have these limitations, and I disagree with forcing line length to 80 characters max with no discussion. Even PEP8 [agrees that you can increase line length to 99 if you want](https://peps.python.org/pep-0008/#maximum-line-length).

Or, if we're **really** firm on this, we should probably fix the default MD index file so it complies with this.

Reply:
I asked in the [art-of-documentation meeting](https://docs.google.com/document/d/13uKW8CC09eecLGkb8toEM_qx1RP3C2riPNSH_aElQBU/edit?tab=t.0#bookmark=id.ln2m5ydzmre5) what the blue team felt about this and they were largely on the side of dropping this rule. If you could ask in the green meeting that would be great, @k-dimple
I think if the green team disagrees and wants this rule in place we could implement it with 120, or if anyone is really insistent on a shorter length than that we could compromise with 100 :)

Reply:
I think we should limit the line length, as I’ve seen extreme cases where people have one-liner markdown docs with lengths of 200-300 characters. We can start with a more flexible guideline of 120 or even 150, but we should put some limit.